ruby/src/api.cc

```diff
@@ -136,11 +136,7 @@ namespace leatherman { namespace ruby {
         } else {
             LOG_INFO("ruby was already loaded.");
         }
-        try {
-            return unique_ptr<api>(new api(move(library)));
-        } catch (lth_lib::missing_import_exception& ex) {
```

Comment:
Erm, never mind, for some reason it was being rethrown (incorrectly too).  Did this return `nullptr` at one point?